File: Teledyne DALSA/Sapera/Demos/NET/MultiBoardSyncGrabDemo/CSharp/MultiBoardSyncGrabDemo.cs
using System;
using System.Collections.Generic;
using System.Windows.Forms;


namespace DALSA.SaperaLT.Demos.NET.CSharp.MultiBoardSyncGrabDemo
{
    static class MultiBoardSyncGrabDemo
    {
        /// <summary>
        /// The main entry point for the application.
        /// </summary>
        [STAThread]
        static void Main()
        {
            Application.EnableVisualStyles();
            Application.SetCompatibleTextRenderingDefault(false);
            MultiBoardSyncGrabDemoDlg form = new MultiBoardSyncGrabDemoDlg();
            if (!form.IsDisposed)
                Application.Run(form);
        }
    }
}

File: Teledyne DALSA/Sapera/Demos/NET/MultiBoardSyncGrabDemo/VB/MultiBoardSyncGrabDemo.vb
Public Class MultiBoardSyncGrabDemo
   'The main entry point for the application.
   <STAThread()> _
   Shared Sub Main()
      Application.EnableVisualStyles()
      Application.SetCompatibleTextRenderingDefault(False)
      Dim MultiBoardSyncGrabDemo As MultiBoardSyncGrabDemoDlg = New MultiBoardSyncGrabDemoDlg
      If Not MultiBoardSyncGrabDemo.IsDisposed Then
         Application.Run(MultiBoardSyncGrabDemo)
      End If
   End Sub
End Class
